src/content/en/2021/cdn.md

```diff
@@ -17,4 +17,420 @@ featured_stat_3: TODO
 featured_stat_label_3: TODO
```

Comment:
Could you add a bio for yourself and also include featured quote/stats? See the [wiki](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters) for more info.

Please also add `unedited: true`
```suggestion
featured_stat_label_3: TODO
unedited: true
```